packages/manager/apps/sign-up/src/form/form.html

```diff
@@ -75,4 +75,8 @@
             </oui-step-form>
         </oui-stepper>
     </ovh-sign-up-form>
+    <div data-ng-if="!$ctrl.isIndiaSubsidiary" class="mt-2">
+        <span class="mandatory-fields-star">*</span>
+        <i data-translate="sign_up_form_mandatory_fields"></i>
+    </div>
```

Comment:
Why don't we display this to Indian customer ?
I don't see any request for this behavior in the ticket

Reply:
Thanks, I forgot to delete it!